components/decimal/src/options.rs

```diff
@@ -7,6 +7,7 @@
 /// A bag of options defining how numbers will be formatted by
 /// [`FixedDecimalFormat`](crate::FixedDecimalFormat).
 #[derive(Debug, Eq, PartialEq, Clone, Default)]
+#[allow(clippy::exhaustive_structs)] // this type is stable
```

Comment:
Nope. Options bags should be `non_exhaustive`. This was one of the primary motivators.